sdk/tests/deqp/functional/gles3/es3fFragmentOutputTests.js

```diff
@@ -875,7 +873,7 @@ var tcuImageCompare = framework.common.tcuImageCompare;
 
                     finalThreshold = tcuTextureUtil.select(
                                     deMath.max(formatThreshold, [precThreshold, precThreshold, precThreshold, precThreshold]),
-                                    [1, 1, 1, 1], // C++ version: UVec4(~0u) bitwise not, all bits in the integer will be flipped
+                                    [0xffffffff, 0xffffffff, 0xffffffff, 0xffffffff], // C++ version: UVec4(~0u) bitwise not, all bits in the integer will be flipped
```

Comment:
https://android.googlesource.com/platform/external/deqp/+/deqp-dev/modules/gles3/functional/es3fFragmentOutputTests.cpp#921
~0 should be 0xffffffff.